apps/internal/oauth/ops/internal/comm/comm.go

```diff
@@ -255,6 +255,7 @@ func (c *Client) do(ctx context.Context, req *http.Request) ([]byte, error) {
 	switch reply.StatusCode {
 	case 200, 201:
 	default:
+		reply.Body = ioutil.NopCloser(bytes.NewBuffer(data))
```

Comment:
Can this be moved to before the `switch` statement?  If this block is ever refactored it would be easy to accidentally miss setting `reply.Body`.

Reply:
Resolved in this commit : https://github.com/AzureAD/microsoft-authentication-library-for-go/pull/178/commits/da3abaa86143db33a44497c61f65f67cc79e6872